components/odh-notebook-controller/Makefile

```diff
@@ -2,8 +2,8 @@
 include makefile-vars.mk
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/opendatahub/odh-notebook-controller
-TAG ?= $(shell git describe --tags --always)
+IMG ?= quay.io/rh_ee_atheodor/odh-notebook-controller
+TAG ?= runtime-configmapv1
```

Comment:
should this tag be merged on main?
I ask because I got the impression that the tag here and params.env should be changed only on the feature branches, not on main one.
asking just to confirm if I got it right or if I missed something.

Reply:
I see your comment on the PR about reverting them, no problem then, thanks